src/gui/editors/AutomationEditor.cpp

```diff
@@ -139,6 +139,9 @@ AutomationEditor::AutomationEditor() :
 	}
 	m_quantizeModel.addItem( "1/192" );
 
+	connect(&m_quantizeModel, SIGNAL(dataChanged()), this, SLOT(setQuantization()));
+	m_quantizeModel.setValue( m_quantizeModel.findText( "1/4" ) );
```

Comment:
I've been using the automation editor lately, and find the 1/8 quantization to work best for my needs, tbqh. 1/4 feels too limiting and 1/16 is too much.